junction/profiles/models.py

```diff
@@ -1,3 +1,17 @@
-# from django.db import models
+from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+
+class Profiles(models.Model):
+	''' 
+	It stores the City/Phone Details of the User.
+	'''
+	user=models.OneToOneField(User)
+	city=models.CharField(max_length=100, blank=False, null=False)
+	contact_no = models.CharField(max_length=10, blank=False, null=False)
+
+	created = models.DateTimeField("Created", null=True, auto_now_add=True)
```

Comment:
Don't add `created/modified`, there is already mixins for this. Have a look at `junction/base/models.py`. `AuditModel`.